index.html

```diff
@@ -182,43 +242,87 @@ <h1 id="t_about">About me</h1>
 
       <section id="more_about">
 
-          <ul id="style1" class="laguage">
-            <li class="align">Language
-              <ul id="list" class="none">
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>Html</li>
-                <li>Css</li>
-              </ul>
-            </li>
-          </ul>
-        
-          <ul id="style2" class="frameworks">
-            <li class="align">Frameworks
-              <ul id="list" class="none">
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-              </ul>
-            </li>
-          </ul>
+        <ul id="style1" class="laguage">
+          <li class="align">Language
+            <ul id="list" class="none">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </li>
+        </ul>
 
-          <ul id="style3" class="skills">
-            <li class="align">Skills
-              <ul id="list" class="none">
-                <li clas>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-              </ul>
-            </li>
-          </ul>
+        <ul id="style2" class="frameworks">
+          <li class="align">Frameworks
+            <ul id="list" class="none">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </li>
+        </ul>
+
+        <ul id="style3" class="skills">
+          <li class="align">Skills
+            <ul id="list" class="none">
+              <li clas>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </li>
+        </ul>
       </section>
     </section>
 
+    <footer id="form_section">
+      <header>
+        <h1 id="last_header">I'm always interested in hearing about new projects, so if you'd like to chat please get in
```

Comment:
[OPTIONAL] Do not use more than one h1 tag on a page. Usually, pages will have a single h1 heading and including two might make search engines think this as an attempt to include more keywords for SEO in multiple h1 tags. It is better to divide the content into two separate topics on individual pages with their own h1 tags.

Reply:
I knew about that, but last week, in my previous project, someone asked me to add another h1 tag for the title "about me". That wasn't very clear to me because I had my first h1 tag in the title, but you are the second person who told me that it should have just one h1 tag by page, Thank you.

Reply:
Fixed.